cmake/KokkosCTest.cmake.in

```diff
@@ -233,8 +233,8 @@ endforeach()
 
 # capture submit error so it doesn't fail because of a submission error
 ctest_submit(RETURN_VALUE submit_ret
-    RETRY_COUNT 2
-    RETRY_DELAY 10
+    RETRY_COUNT 5
+    RETRY_DELAY 60
```

Comment:
Elaborate on why this is is needed.  This means stalling machines for 5min if the cdash server is unresponsive.  This is not negligible.